tests/src/kakarot/test_state.py

```diff
@@ -89,8 +89,8 @@ class TestCachePreaccessedAddresses:
         @SyscallHandler.patch("IERC20.balanceOf", lambda addr, data: [0, 1])
         def test_should_cache_precompiles(self, cairo_run):
             state = cairo_run("test__cache_precompiles")
-            assert list(state["accounts"].keys()) == [
-                f"0x{i:040x}" for i in range(1, 10)
+            assert list(map(str.lower, state["accounts"].keys())) == [
+                f"0x{i:040x}" for i in range(1, 11)
```

Comment:
easier to just do `f"0x{i:040x}".upper()`
but the truth is to use `to_checksum_address`

Reply:
done